runtime/ftplugin/zig.vim

```diff
@@ -1,44 +1,44 @@
 " Vim filetype plugin file
 " Language: Zig
-" Upstream: https://github.com/ziglang/zig.vim
 
-" Only do this when not done yet for this buffer
 if exists("b:did_ftplugin")
   finish
 endif
 
 let b:did_ftplugin = 1
 
-let s:cpo_orig = &cpo
+let s:cpo_save = &cpo
 set cpo&vim
 
 compiler zig_build
 
 " Match Zig builtin fns
 setlocal iskeyword+=@-@
-
-" Recommended code style, no tabs and 4-space indentation
-setlocal expandtab
-setlocal tabstop=8
-setlocal softtabstop=4
-setlocal shiftwidth=4
-
 setlocal formatoptions-=t formatoptions+=croql
-
-setlocal suffixesadd=.zig,.zir
+setlocal suffixesadd=.zig,.zir,.zon
+let &l:define='\v(<fn>|<const>|<var>|^\s*\#\s*define)'
+let b:undo_ftplugin = 'setl isk< fo< sua< mp< def<'
+
+if get(g:, 'zig_recommended_style', 1)
```

Comment:
> if get(g:, 'zig_recommended_style', 1)

That variable should be documented. Please place a short section into runtime/doc/syntax.txt (e.g. see how it is done for other synax files, like `:h ft-python-syntax` . Don't forget to add a tag and update the last change header please.

Reply:
What is meant by tag and last change header?

Reply:
Basically add a help jump target, like it is done now: `*g:zig_recommended_style*` in the help file. And then we usually have a separate line, mentioning when the last change was done, like here: https://github.com/vim/vim/blob/master/runtime/ftplugin/c.vim#L4